notebooks/exploratory/sdat2-A-plot-style-test.ipynb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
<p>Looks neat - very handy scaling functions for the sizes and I like the (a), (b) , .... labelling! ! </p><p> </p><p>Could you give an example using the color palette as well to see the colors? (:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai4er-cdt/gtc-biodiversity/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
sure, will do :)

Reply:
kind of implemented